### ISSUE TITLE
EventDataDeserializationException

### ISSUE THREAD

**Issue by andrioni** [shyiko/mysql-binlog-connector-java#69]
Hi, I'm getting this exception whenever I try to read a mutation event from a binlog:

```
com.github.shyiko.mysql.binlog.event.deserialization.EventDataDeserializationException: Failed to deserialize data of EventHeaderV4{timestamp=1445568293000, eventType=EXT_WRITE_ROWS, serverId=1, headerLength=19, dataLength=49, nextPosition=560, flags=0}
	at com.github.shyiko.mysql.binlog.event.deserialization.EventDeserializer.deserializeEventData(EventDeserializer.java:180)
	at com.github.shyiko.mysql.binlog.event.deserialization.EventDeserializer.nextEvent(EventDeserializer.java:148)
	at com.github.shyiko.mysql.binlog.BinaryLogFileReader.readEvent(BinaryLogFileReader.java:82)
...
```

The binlog file I'm trying to read is available [in this gist](https://gist.github.com/andrioni/3dd4277435cca42df5d8#file-mysql-bin-000004), and was produced on the `mysql` Docker image running 5.6.26, with row-based replication turned on. I'm using `mysql-binlog-connector-java` 0.2.4.

**Comment by shyiko** [comment]
Hi. Yeah... this is going to be fixed in upcoming 1.0.0 release. Until then you need to specify checksum type (NONE by default) specifically, like so: 

```
EventDeserializer eventDeserializer = new EventDeserializer();
eventDeserializer.setChecksumType(ChecksumType.CRC32);
BinaryLogFileReader binaryLogFileReader = new BinaryLogFileReader(file, eventDeserializer);
...
```

**Comment by andrioni** [comment]
Oh, thank you very much!
